recipes/apache.rb

```diff
@@ -46,3 +48,22 @@
 end
 
 apache_site node['nagios']['server']['vname']
+
+node.default['nagios']['web_user'] = node['apache']['user']
+node.default['nagios']['web_group'] = node['apache']['group'] || node['apache']['user']
+
+# configure the appropriate authentication method for the web server
+case node['nagios']['server_auth_method']
+when 'openid'
```

Comment:
I just moved the code from default.rb to apache.rb, I didn't change the logic (much). It seems pretty harmless, in that the support comes from the apache2 cookbook.